tests/core/pyspec/eth2spec/test/phase_0/epoch_processing/test_process_rewards_and_penalties.py

```diff
@@ -15,13 +15,16 @@
 )
 from eth2spec.test.helpers.attester_slashings import get_indexed_attestation_participants
 from eth2spec.test.phase_0.epoch_processing.run_epoch_process_base import run_epoch_processing_with
+from random import Random
 
 
 def run_process_rewards_and_penalties(spec, state):
     yield from run_epoch_processing_with(spec, state, 'process_rewards_and_penalties')
 
 
-def prepare_state_with_full_attestations(spec, state, empty=False):
+def prepare_state_with_full_attestations(spec, state, participation_fn=None):
```

Comment:
`prepare_state_with_full_attestations` --  "full" no longer seems valid in the name of this function

Reply:
The default is still full, but agree that naming could be better. Suggestions?

Reply:
`prepare_state_with_attestations` and note the default is full in a header comment?